ml-algorithms/src/main/java/org/opensearch/ml/engine/memory/MLMemoryManager.java

```diff
@@ -139,7 +139,7 @@ public void createInteraction(
      * @param actionListener get all the final interactions that are not traces
      */
     public void getFinalInteractions(String conversationId, int lastNInteraction, ActionListener<List<Interaction>> actionListener) {
-        Preconditions.checkArgument(lastNInteraction > 0, "History message size must be at least 1.");
+        // Preconditions.checkArgument(lastNInteraction > 0, "History message size must be at least 1.");
```

Comment:
are we going to keep it like this?

Reply:
Yes, this is for allowing user not use conversation history if they want

Reply:
I mean will we keep this as commented out?

Reply:
fixed in new commit